dashboard/starter-example/app/page.tsx

```diff
@@ -20,7 +21,7 @@ export default function Page() {
             href="/login"
             className="flex items-center gap-5 self-start rounded-lg bg-blue-500 px-6 py-3 text-sm font-medium text-white transition-colors hover:bg-blue-400 md:text-base"
           >
-            <span>Log in</span>
+            <span>Log in</span> <ArrowRightIcon className="w-5 md:w-6" />
```

Comment:
Would this also need to be added to `final-example` too?

Reply:
This is part of the final, we've accidentally removed it from the starter.